RLBotPack/TensorBot/src/bot.cfg

```diff
@@ -0,0 +1,31 @@
+[Locations]
+# Path to loadout config. Can use relative path from here.
+looks_config = ./appearance.cfg
+
+# Path to python file. Can use relative path from here.
+python_file = ./bot.py
+
+# Name of the bot in-game
+name = TensorBot
+
+# The maximum number of ticks per second that your bot wishes to receive.
+maximum_tick_rate_preference = 120
+
+[Details]
+# These values are optional but useful metadata for helper programs
+# Name of the bot's creator/developer
+developer = Soren
+
+# Short description of the bot
+description = 100% Machine Learning bot trained with recorded Diablo data
+
+# Fun fact about the bot
+fun_fact = Takes an additional 5 seconds to load after first starting up
+
+# Link to github repository
+github = https://github.com/DanielDowns/ArtificialIntelligence/tree/master/RocketLeagueBots/TensorBot
+
+# Programming language
+language = python
+
+supports_early_start = True
```

Comment:
This attribute only works if you put it in the [Locations] section like this: https://github.com/tarehart/ReliefBot/blob/f7cd95d5bbc84afa951b7cd62a6a9a71d6f5e100/src/main/python/adversity_bot.cfg#L13